docs/index.md

```diff
@@ -1,97 +1,109 @@
 
-This guide uses a pre-configured application from a template. Application is deployed to a kubernetes cluster. 
-At the end of this tutorial, you should be able to access an working application via VPN and made changes to it.
+This guide walk-through the process of deploying an Sample Java application in CFT.
+
+An application will be generated from a template and deployed to a kubernetes cluster.
+
+At the end of this tutorial, you will be able to access your application via VPN and will have made changes to it.
 
 ## Prerequisite
 
 `Github` access is required to complete the steps in this tutorial. Following the [link](https://hmcts.github.io/onboarding/team/github.html#github) to request access.
 
-## High Level Steps
-
-- Create a Repository and build pipeline
-- Build application
-- Configure load balancing for HA
-- Deploy application
-- Access application
-- Customize application
-- Troubleshooting
 
 ## Steps
 
 #### Create a Repository and build pipeline
 
-1. Access [Backstage](https://backstage.platform.hmcts.net/create) to create a new github repo by selecting predefined [`Spring Boot Service`](https://backstage.platform.hmcts.net/create/templates/springboot-template) template. Please make sure name of  Github repo  starts with `labs*`. This enables Jenkins build tool to organize the folder based on Github repo names.
-
+1. Click `Create` in the Backstage sidebar and select [`Spring Boot Service`](https://backstage.platform.hmcts.net/create) template. Prefix your GitHub repository name with `labs-*`. This enables Jenkins to automatically pickup the folder based on the GitHub repository name.
+   
+   Description and default values for various `Fields` in the template.
+   
+   - Product:  `labs`        #Product this component belongs to, normally the team name, e.g. cmc, labs
+    
+   - Component:            #Name of the component, e.g. backend
+    
+   - Slack contact channel:#Which channel (or user) to contact if there's any issues with this service.
+    
+   - Description:          #Description of the application, a sensible default will be used if not specified
+    
+   - HTTP port:            #The port to run the app on.
+    
+   - GitHub admin team:    #Which GitHub team should have admin permissions, use the format hmcts/<team-id>
+   
+   - Owner:                #Owner of the Component
+   
+   - Host:  `github.com`
+   
+   - Owner:                #The organization, user or project that this repo will belong to
+   
+   - Repository: `labs-<Component>`     #The name of the repository
+   
+    
 2. Create Helm values file with following contents(this is only needed in the lab as the application will be run on sandbox).
 
    CFT: values.sandbox.template.yaml
-   
-   
+
+    ```yaml
         java:
           # Don't modify below here
           image: ${IMAGE_NAME}
           ingressHost: ${SERVICE_FQDN}
+    ```
    
 3. Get the Pull Request reviewed and merged.
```

Comment:
no need to review PR anyway this step should go as you'll move it to the template as in https://github.com/hmcts/golden-path-java/pull/1/files#r756710040